test/jasmine/tests/gl2d_plot_interact_test.js

```diff
@@ -20,6 +20,8 @@ function countCanvases() {
     return d3.selectAll('canvas').size();
 }
 
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 5000;
```

Comment:
Does this apply only to this test suite or to all test suite that get bundled?

Reply:
Good question. I noticed when using shard with high limit number (e.g. 20), changing a timout in one test has an impact on the other tests in another suites. That may be one reason to try to keep them separated. Anyway a better solution may be to have a timeout setup for every suite/describe block to avoid side effects?

Reply:
> Anyway a better solution may be to have a timeout setup for every suite/describe block to avoid side effects?

Yes, that's a better solution.

Reply:
@etpinard @antoinerg Any default value there to start with? Should they be set for every describe block? Or having them in every suites is only required?